base-notebook/start.sh

```diff
@@ -47,7 +47,12 @@ if [ $(id -u) == 0 ] ; then
 
     # Exec the command as NB_USER
     echo "Execute the command: $*"
-    exec su $NB_USER -c "env PATH=$PATH $*"
+    if [ $# -eq 0 ]; then
```

Comment:
I think `start.sh <required command>` was the original intent of the script. If we want to make running bash the default, that's cool. But is the separate sudo form necessary?

Reply:
One could do.

... bash start.sh bash 

This change aims for 
... start.sh
Would arrive at bash.
Right now, it will exit image.

Reply:
I'm OK with that behavior change, but we should also make `bash` the default when `GRANT_SUDO` is not set for consistency.